packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -394,13 +387,15 @@ const checkAccountStatus = (account_status, client, addNotificationMessage, logi
         mt5_withdrawal_locked,
         document_needs_action,
         unwelcome,
-        ukrts_max_turnover_limit_not_set,
```

Comment:
Clarification:
What is `ukrts` short for?

Reply:
tbh im not really sure of what its short for. never saw this status before, as discussed with @negar-binary `ukrts_max_turnover_limit_not_set` is deprecated status